components/company/Hero.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+  export interface CompanyInfo {
+    name: string
+    description: string
+    logo: string
+    website: string
+  }
+
+  const company = defineProps({
+    content: {
+      type: Object as PropType<CompanyInfo>,
+      required: true,
+    },
+  })
+
+  const openNewTab = () => {
+    navigateTo(company.content.website, {
+      external: true,
+      open: {
+        target: '_blank',
+      },
+    })
+  }
+</script>
+
+<template>
+  <div>
+    <VSheet class="w-75">
+      <h2
+        class="text-md-h2 text-sm-h3 text-h4 text-center pb-10 font-weight-bold"
+      >
+        Hovedpartner
+      </h2>
+
+      <VRow class="pt-10">
+        <!-- Company logo -->
+        <VCol cols="12" lg="6" style="cursor: pointer">
+          <VHover v-slot="{ isHovering, props }">

```

Comment:
The hover effect here looks nice, but we will only have rectangular, full-covering logos, in a few instances. I recommend looking into the difference between `box shadow`, and `drop shadow` using plain SCSS instead of the vuetify hover components. 

NB! If I remember correctly, not all browsers support `drop shadow`, so this might be a potential trade off.

Here's an example of what I'm trying to say
![image](https://github.com/et-dagen/et-dagen/assets/26115581/c08ea87b-6fde-42eb-8485-9fe011e61db4)